code/__DEFINES/maths.dm

```diff
@@ -91,6 +91,9 @@
 // Returns the nth root of x.
 #define ROOT(n, x) ((x) ** (1 / (n)))
 
+// Real modulus that handles decimals
+#define ROLL_DICE(amt, faces) (rand(0, (faces-1)*amt) + amt)
```

Comment:
This is not how you roll dice. The probability is evenly distributed and it shouldn't be.

Easiest fix I can think of would be to outsource this to some nice old lady in China for $1 a day.

Reply:
We dont use this anymore